package.json

```diff
@@ -1,54 +1,38 @@
 {
   "name": "remotely-signed-s3",
-  "version": "3.2.0",
+  "version": "4.0.0",
   "description": "Do remote signing for S3 using V4 signatures",
   "main": "lib/index.js",
   "dependencies": {
-    "aws4": "^1.6.0",
-    "babel-runtime": "^6.23.0",
-    "debug": "^2.6.8",
-    "joi": "^10.6.0",
+    "aws4": "^1.8.0",
+    "debug": "^4.1.1",
+    "joi": "^14.3.1",
     "libxmljs": "^0.19.5",
-    "mz": "^2.6.0",
-    "tmp": "^0.0.31"
+    "mz": "^2.7.0",
+    "tmp": "^0.0.33"
   },
   "devDependencies": {
-    "assume": "^1.4.1",
-    "babel-compile": "^2.0.0",
-    "babel-preset-taskcluster": "^3.0.0",
-    "mocha": "^3.4.2",
-    "sinon": "^2.3.8",
-    "source-map-support": "^0.4.11",
+    "assume": "^2.1.0",
+    "mocha": "^5.2.0",
+    "sinon": "^7.2.2",
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "compile": "babel-compile -p taskcluster src:lib test-src:test",
-    "prepublish": "npm run compile",
-    "pretest": "npm run compile",
     "test": "DEBUG= mocha $MOCHA_FLAGS"
   },
+  "engines": {
+    "node": ">=10.0.0",
```

Comment:
Why 10? Does this use things not available in 8?

Reply:
I've added 8 back, it shouldn't use anything 10 specific.